### ISSUE TITLE
Allow reversing mobile swipe direction 

### ISSUE THREAD

**Issue by Kalidibus** [rr-/szurubooru#562]
When using a smartphone the direction to swipe in is the opposite of what most other apps use. Is there a way to reverse the direction?

Example: when browsing through posts I swipe left (as if I was pushing the current image to the left) I expect this to take me to the next post which on the main screen is situated to the right. Instead it takes me to the previous post.

**Comment by parallax4** [comment]
You're right, but, in my opinion, it shouldn't even be an option and should just be the standard behavior.

**Comment by Kalidibus** [comment]
https://github.com/rr-/szurubooru/blob/d7d2a151a8d60a8bde87a8245da299cd6489552f/client/js/views/post_main_view.js#L104-L116

I'm not sure if this is the section causing the issue. Is it as simple as reversing these two?

**Comment by parallax4** [comment]
@Kalidibus That would be a simple way to change the behavior, but probably the most correct/clear way to change it would be to swap these: https://github.com/rr-/szurubooru/blob/d7d2a151a8d60a8bde87a8245da299cd6489552f/client/js/views/post_main_view.js#L106-L114

**Comment by wsippel** [comment]
So I'm not the only one - the current implementation feels backwards. Also, it seems to conflict with pinch gestures, at least on iPad. No idea if that can be fixed, but if it can't, maybe the entire touch function should a setting, so users could disable it entirely?

**Comment by negiworx** [comment]
I agree with the idea of having a setting to enable disable it. I keep having it move to another post when trying to look at an image that isn't scaled to the size of the screen.
